PyFlyt/models/target.urdf

```diff
@@ -15,7 +15,7 @@
 
     <visual>
       <geometry>
-        <sphere radius="0.05"/>
+        <sphere radius="1"/>
```

Comment:
Don't modify this file.

Reply:
Reverted

Reply:
The `FWTargets.py` env has the radius as 2 meters, is it acceptable to leave this file as 0.05m, since it is only for visuals?

Reply:
Ideally make a new urdf for the big targets.